src/main/java/pack/controller/PersonController.java

```diff
@@ -0,0 +1,68 @@
+package pack.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import pack.model.Person;
+import pack.service.IPersonService;
+import pack.util.CustomException;
+import pack.util.ResponseFormat;
+
+@Controller
+public class PersonController {
+
+	private HttpClient httpClient;
+	private HttpGet request;
+	private HttpResponse response;
+	private ObjectMapper objectMapper;
+
+	@Autowired
+	private IPersonService personService;
+
+	@PostConstruct
+	public void init() {
+		httpClient = HttpClientBuilder.create().build();
+		request = new HttpGet("https://jsonplaceholder.typicode.com/users");
+		objectMapper = new ObjectMapper();
```

Comment:
Ok, please refactor and don't use object mapper at all. Instead return a ResponseData<YourResponseObject>. Your response object should contain those 3 fields, status, data, and reason. Check ResponseData from spring.

Reply:
done